packages/esm-patient-medications-app/src/order-basket/order-basket-search-results.component.tsx

```diff
@@ -32,12 +33,20 @@ export default function OrderBasketSearchResults({
 
   useEffect(() => {
     const abortController = new AbortController();
-    searchMedications(searchTerm, encounterUuid, abortController, config.daysDurationUnit).then(
-      setSearchResults,
-      createErrorHandler,
-    );
+    if (searchTerm) {
+      if (searchTerm.length == 1) {
```

Comment:
If this is intended to minimize the number of times we hit the server, we should consider debouncing the search instead (possibly in a follow-up PR).

Reply:
Nope, @denniskigen the minimum requirement in terms of the length of a `searchTerm` by the server is 2. Other words to expect non empty result(s) from the server, the length of the search should be greater that 1. But that doesn't nullify your suggestion, we definitely need to debounce..